examples/remote-view/index.mjs

```diff
@@ -7,7 +7,7 @@ import * as fs from 'fs';
 const stdlib = loadStdlib(process.env);
 
 (async () => {
-  if ( stdlib.connector === 'ALGO' ) { return; }
+  if ( ['ALGO', 'CFX'].includes(stdlib.connector) ) { return; }
```

Comment:
I think my preference is to continue running it on CFX, but add it to xfail, so we don't forget about the error by losing it in some old Jira card in the backlog.

Reply:
Looks like it was added to xfail two days ago.. I'll close this PR